lib/convection/model/cloudfile.rb

```diff
@@ -14,6 +14,7 @@ module Cloudfile
       attribute :name
       attribute :region
       attribute :splay
```

Comment:
We wont need it after we remove it from converge/delete. I am only doing these changes for diff as the testing effort is quite substantial. I will do converges once finished with diffs.